_includes/_help/search/_posts/second-order.md

```diff
@@ -1,4 +1,7 @@
-We provide five second-order operators which modify the query results by performing second-order operations on the original query. To invoke the operators, enter the corresponding operator before your search terms (enclosing your search terms in parentheses) in the search box. The search terms enclosed by parentheses are considered the first-order query.
+We provide five second-order operators which modify the query results by performing second-order operations on the original query. Theese operators are database functions which form secondary queries based on attributes of the objects returned in an initial query; they can provide powerful methods to investigate complex, multipartite information graphs. To invoke the operators, enter the corresponding operator before your search terms (enclosing your search terms in parentheses) in the search box. The search terms enclosed by parentheses are considered the first-order query.
+
+In addition to the basic information provided below, we have [published a tutorial describing these operators in detail](/abs/2020BAAS...52.0207K/abstract), both alone and in conjunction with other functions. It is intended for scientists and others who wish to make fuller use of the SciX database. 
```

Comment:
In the link, you'll need to add the domain, but with an `if` to switch between ADS and SciX, like this:
`{% if include.ads %}{{ site.ads_base_url }}{% else %}{{ site.scix_base_url }}{% endif %}/abs/2020BAAS...52.0207K/abstract`